<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomDialog</class>
 <widget class="QDialog" name="RoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>会议室</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
    background-color: #222222;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: #2b2b2b;
    border-bottom: 1px solid #444444;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_title">
        <property name="styleSheet">
         <string notr="true">color: white;
font-size: 16px;
font-weight: 500;</string>
        </property>
        <property name="text">
         <string>房间号：123456</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_quit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #dc3545;
    color: white;
    border: none;
    border-radius: 6px;
    font-size: 14px;
    padding: 8px 16px;
}

QPushButton:hover {
    background-color: #c82333;
}

QPushButton:pressed {
    background-color: #bd2130;
}</string>
        </property>
        <property name="text">
         <string>退出房间</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainContent" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="videoArea" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: #1a1a1a;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="UserShow" name="wdg_userShow" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: black;
border: 2px solid #444444;
border-radius: 8px;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QWidget" name="danmakuContainer" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>120</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea {
    background-color: #2b2b2b;
    border: none;
    border-left: 1px solid #444444;
}

QScrollBar:vertical {
    background-color: #2b2b2b;
    width: 10px;
}

QScrollBar::handle:vertical {
    background-color: #555555;
    border-radius: 5px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background-color: #666666;
}</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>218</width>
           <height>578</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="wdg_list" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: transparent;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: #2b2b2b;
    border-top: 1px solid #444444;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QComboBox" name="cb_moji">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    background-color: #3a3a3a;
    color: white;
    border: 1px solid #555555;
    border-radius: 6px;
    padding: 8px 12px;
    font-size: 14px;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    width: 0; 
    height: 0; 
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
    border-top: 5px solid white;
}

QComboBox QAbstractItemView {
    background-color: #3a3a3a;
    color: white;
    selection-background-color: #007AFF;
    border: 1px solid #555555;
}</string>
        </property>
        <item>
         <property name="text">
          <string>无滤镜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>兔耳朵</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>帽子</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_audio">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    color: white;
    font-size: 14px;
    background-color: #3a3a3a;
    border: 1px solid #555555;
    border-radius: 6px;
    padding: 0 15px;
}

QCheckBox:hover {
    background-color: #4a4a4a;
}

QCheckBox:checked {
    background-color: #007AFF;
    border-color: #007AFF;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}</string>
        </property>
        <property name="text">
         <string>🎤 麦克风</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_video">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    color: white;
    font-size: 14px;
    background-color: #3a3a3a;
    border: 1px solid #555555;
    border-radius: 6px;
    padding: 0 15px;
}

QCheckBox:hover {
    background-color: #4a4a4a;
}

QCheckBox:checked {
    background-color: #007AFF;
    border-color: #007AFF;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}</string>
        </property>
        <property name="text">
         <string>📹 摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_desk">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    color: white;
    font-size: 14px;
    background-color: #3a3a3a;
    border: 1px solid #555555;
    border-radius: 6px;
    padding: 0 15px;
}

QCheckBox:hover {
    background-color: #4a4a4a;
}

QCheckBox:checked {
    background-color: #007AFF;
    border-color: #007AFF;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}</string>
        </property>
        <property name="text">
         <string>🖥️ 屏幕共享</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Recognition">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    color: white;
    font-size: 14px;
    background-color: #3a3a3a;
    border: 1px solid #555555;
    border-radius: 6px;
    padding: 0 15px;
}

QCheckBox:hover {
    background-color: #4a4a4a;
}

QCheckBox:checked {
    background-color: #007AFF;
    border-color: #007AFF;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}</string>
        </property>
        <property name="text">
         <string>📝 字幕</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_close">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #dc3545;
    color: white;
    border: none;
    border-radius: 6px;
    font-size: 14px;
    padding: 8px 16px;
}

QPushButton:hover {
    background-color: #c82333;
}

QPushButton:pressed {
    background-color: #bd2130;
}</string>
        </property>
        <property name="text">
         <string>挂断</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserShow</class>
   <extends>QWidget</extends>
   <header>usershow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
